src/modules/job-info/job_state.c

```diff
@@ -425,6 +456,75 @@ static int jobspec_parse (struct info_ctx *ctx,
         assert (job->job_name);
     }
 
+    if (json_unpack_ex (jobspec, &error, 0,
+                        "{s:o}",
+                        "resources", &resources) < 0) {
+        flux_log (ctx->h, LOG_ERR,
+                  "%s: job %llu invalid jobspec: %s",
+                  __FUNCTION__, (unsigned long long)job->id, error.text);
+        goto error;
+    }
+
+    /* Set job->task_count
+     */
+    if (json_unpack_ex (tasks, NULL, 0,
+                        "[{s:{s:i}}]",
+                        "count", "total", &job->task_count) < 0) {
+        int per_slot, slot_count = 0;
+        struct res_level res[3];
+
+        if (json_unpack_ex (tasks, NULL, 0,
+                            "[{s:{s:i}}]",
+                            "count", "per_slot", &per_slot) < 0) {
+            flux_log (ctx->h, LOG_ERR,
+                      "%s: job %llu invalid jobspec: %s",
+                      __FUNCTION__, (unsigned long long)job->id, error.text);
+            goto error;
+        }
+        if (per_slot != 1) {
+            flux_log (ctx->h, LOG_ERR,
+                      "%s: job %llu: per_slot count: expectedc 1 got %d: %s",
```

Comment:
just noticed typo here `expectedc`

Reply:
though probably the typo existed in the original code cut-and-pasted here.

Reply:
nah, i seem to have added the extra c by myself :P